### PR TITLE
Rebase/element android v1.4.2

### PR DESCRIPTION
**Pull request by Florian14**
#460

### CODE REVIEW

File: vector/src/main/res/values/config.xml [hunk 1]
```diff
@@ -7,9 +7,8 @@
     <string name="matrix_org_server_url" translatable="false">https://matrix.org</string>
 
     <!-- Rageshake configuration -->
-    <string name="bug_report_url" translatable="false">https://riot.im/bugreports/submit</string>
```
Comment (yostyle) [inline vector/src/main/res/values/config.xml:9]: please restore this line and comment it

File: vector/src/main/res/values/config.xml [hunk 2]
```diff
@@ -30,14 +29,14 @@
...
-<!--    </string-array>-->
+    <!--    <string-array name="permalink_supported_hosts" translatable="false">-->
+    <!--        &lt;!&ndash; Regular Element Web instance &ndash;&gt;-->
+    <!--        <item>app.element.io</item>-->
+    <!--        &lt;!&ndash; Other known instances of Element Web &ndash;&gt;-->
+    <!--        <item>develop.element.io</item>-->
+    <!--        <item>staging.element.io</item>-->
+    <!--        &lt;!&ndash; Previous Web instance, kept for compatibility reason &ndash;&gt;-->
+    <!--        <item>riot.im</item>-->
+    <!--    </string-array>-->
```
Comment (yostyle) [inline vector/src/main/res/values/config.xml:40]: Please undo these changes

File: vector/src/main/res/values/config.xml (hunk 1 above)
Comment (yostyle) [inline vector/src/main/res/values/config.xml:9]: please restore this line and comment it

File: vector/src/main/res/values/config.xml [hunk 3]
```diff
@@ -19,7 +18,7 @@
     <!-- Note: pusher_http_url should have path '/_matrix/push/v1/notify' -->
     <string name="pusher_http_url" translatable="false">https://matrix.org/_matrix/push/v1/notify</string>
```
Comment (yostyle) [inline vector/src/main/res/values/config.xml:19]: `<string name="pusher_http_url" translatable="false">https://sygnal.tchap.gouv.fr/_matrix/push/v1/notify</string>`

**Review by giomfo — APPROVED** [review]
SGTM
